scenarioo-server/src/main/java/org/scenarioo/rest/configuration/FlatLabelConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package org.scenarioo.rest.configuration;
+
+import org.scenarioo.model.configuration.LabelConfiguration;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
```

Comment:
why do we need  the root element annotation?

Reply:
It does not seem to be necessary. I removed it.